packages/w3up-client/src/client.js

```diff
@@ -291,9 +292,66 @@ export class Client extends Base {
         )
       }
     }
+
     return space
   }
 
+  /**
+   * Authorizes a gateway to serve content from the provided space and record egress events.
+   * Delegates the following capabilities to the gateway:
+   * - `space/content/serve/*`
+   *
+   * @param {import('./types.js').OwnedSpace} space - The space to authorize the gateway for.
+   * @param {object} [options] - Options for the authorization.
+   * @param {`did:web:${string}`} [options.gateway] - The Web DID of the gateway to authorize. If not provided, the default `did:web:w3s.link` gateway will be authorized.
+   * @param {number} [options.expiration] - The time in seconds to expire the authorization.
```

Comment:
Not quite, this is the time at which the delegation expires in seconds from unix epoch.

Reply:
You probably need some connection config here right?